### DISCUSSION TITLE
Understanding declines and failed payments

### DISCUSSION THREAD

**Discussion by superstar1205** [Web-development-s/Stripe-with-laravel#46] [Q&A] [answered]
Learn why some payments fail and what you can do to decrease your decline rate.

**Comment by VitaminC1211** [comment]
Payments can fail for a variety of reasons and it’s frustrating when they result in the loss of legitimate business. Many payments fail for good reason and do so to minimize the possibility of a fraudulent payment.

We continuously work to reduce decline rates for cards processed on Stripe with machine learning and experimentation, along with issuer and network outreach to optimize acceptance rates for all Stripe businesses—often without requiring any changes to your integration.

**Comment by superstar1205** [comment]
Good!

**Comment by dev0115** [comment, marked as answer]
Great!
